kong/cmd/restart.lua

```diff
@@ -0,0 +1,25 @@
+local conf_loader = require "kong.conf_loader"
+local stop = require "kong.cmd.stop"
+local start = require "kong.cmd.start"
+
+local function execute(args)
+  local conf = assert(conf_loader(args.conf))
+  args.prefix = conf.prefix -- Required for stop
+  args.graceful = nil -- Restart is always not graceful (reload should be used instead)
```

Comment:
What is the purpose of this? There is no `graceful` option in the `lapp` string so none can be given anyways.